.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,13 +25,14 @@ Fixes # (issue)
 - [ ] Test A
 - [ ] Test B
 
+## Mandatory Tasks
+- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.
```

Comment:
Moves the self-review guideline to a new place so that it's noticed immediately. Also, added under a stricter headline.